sql_schema/BookMyShow.sql

```diff
@@ -0,0 +1,50 @@
+CREATE DATABASE Movies;
+
+USE Movies;
+
+CREATE TABLE Customers(
+CustomerID INT PRIMARY KEY,
```

Comment:
Do we need this this many info, while booking tickets ? like phone, email, name ?